### DISCUSSION TITLE
If your Nightscout only has 10GB of disk space, ...

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2707] [Show and tell]
If you set up Nightscout on Google Cloud a long time ago with a 10GB disk space, please follow the instructions to increase disk size to 30GB:
https://navid200.github.io/xDrip/docs/Nightscout/FullDisk.html

**Comment by Navid200** [comment]
Please increase the disk size before your system stops working.  If you wait until your system dies, the resolution is more complicated.
It's all explained here:  https://navid200.github.io/xDrip/docs/Nightscout/FullDisk.html

**Comment by AlJeske** [comment]
This is exactly what happened to me, no menu (connecting to serial ports disabled) I have already been using Xdrip+ and want to be able to show my Endocrinologist the data. Setup is: Dexcom G6=>Xdrip+=>Aaps. If there is a better way for the Endo to see my progress, let me know and I will forego the Nightscout-google cloud endeavor. No Menu When I click SSH.
Sorry I didn't know how to start the discussion.  I hope this is the right place.

**Reply by Navid200** [reply]
Please open a new discussion.
This discussion is clearly about disk space.

**Reply by AlJeske** [reply]
ok I'll try
